README.md

```diff
@@ -12,9 +12,9 @@ Run the following commands on an all-snap image:
 
     $ sudo snap install --beta --devmode classic
     $ sudo classic
-    (classic)ubuntu@localhost:~$ sudo apt update
```

Comment:
Why dropping it? Its there to show that apt is available in the classic environment.